server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -132,6 +133,14 @@ const orderMutations = {
         type: PaymentMethodReferenceInput,
         description: 'Reference to a Payment Method to update the order with',
       },
+      tier: {
+        type: TierReferenceInput,
+        description: 'Reference to a Tier to update the order with',
+      },
+      amount: {
+        type: GraphQLInt,
+        description: 'Reference to an Amount to update the order to',
```

Comment:
It's not a Reference, it's an amount.

I think for better API design, we need to replace `GraphQLInt` with a new `AmountInput` type.